db/setup.js

```diff
@@ -0,0 +1,32 @@
+const { Pool, Client } = require('pg')
```

Comment:
Si on veut ajouter des tables plus tard on mise uniquement sur "IF NOT EXISTS" ? 
Il y a des frameworks d'évolutions de table pour node ?

Reply:
Par exemple, https://github.com/db-migrate/node-db-migrate

Ah voir si ça vaut le coup, je travaille avec ce genre de chose en Scala, c'est dans mon framework

Reply:
Je suis aussi habitué aux migrations DB avec Django. J'ai regardé node-db-migrate mais le projet n'est pas très maintenu (spécialement le driver PG qui va avec).

J'ai regardé les libs node à ce sujet et je pense que [KnexJS](https://github.com/knex/knex) serait très utile. C'est un query-builder (pas un ORM entier) qui gère aussi les migrations. J'ai mis un message sur Slack pour en discuter avec les autres aussi.